ga4gh/datamodel/variants.py

```diff
@@ -490,9 +506,30 @@ def _addFile(self, filename):
                 if chrom in self._chromFileMap:
                     raise Exception("cannot have overlapping VCF/BCF files.")
                 self._updateMetadata(varFile)
+                self._updateCallSetIds(varFile)
                 self._chromFileMap[chrom] = varFile
 
-    def convertVariant(self, record):
+    def _convertGaCall(self, recordId, name, call):
+        c = protocol.GACall()
```

Comment:
in other similar places in the code we've used a lowercased entity name (minus the "GA") for the variable,
for example 
```
variant = protocol.GAVariant()
```
Thus I'd recommend  naming this `call` instead of `c`.